locales/typed/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "typed",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
```

Comment:
Add `"private": true`

Reply:
updated in: [6600895](https://github.com/scpwiki/wikijump/pull/2031/commits/66008951c1c5a265a8aacb22469beeadf01d26f4)